senders/discord/init.go

```diff
@@ -93,7 +96,11 @@ func (sender *Sender) runBot() {
 	worker.NewWorker(
 		workerName,
 		sender.logger,
-		sender.DataBase.NewLock(discordLockName, discordLockTTL),
+		sender.DataBase.NewLock(discordLock(contactType), discordLockTTL),
 		workerAction,
 	).Run(nil)
 }
+
+func discordLock(name string) string {
```

Comment:
Тогда тут можно будет так

```suggestion
func discordLockKey(name string) string {
```